flickit-assessment-kit/src/main/java/org/flickit/assessment/kit/adapter/out/persistence/kitversion/KitVersionMapper.java

```diff
@@ -35,4 +36,8 @@ public static KitVersion mapToDomainModel(KitVersionJpaEntity entity) {
             entity.getCreatedBy(),
             entity.getLastModifiedBy());
     }
+
+    public static long mapKitVersionToVersionStatus(KitVersionStatus status, long id){
+        return status == KitVersionStatus.ARCHIVE ? -id : status.getId();
+    }
```

Comment:
**Suggestion:** Modify the `mapKitVersionToVersionStatus` method to handle all potential `KitVersionStatus` values to ensure that the method does not return unexpected results or errors. [general, importance: 6]
```suggestion
    public static long mapKitVersionToVersionStatus(KitVersionStatus status, long id){
        switch (status) {
            case ARCHIVE:
                return -id;
            case ACTIVE:
                return id + 1000;  // Example adjustment
            default:
                return status.getId();
        }
    }
```